frontend/server/src/Request.php

```diff
@@ -146,6 +146,37 @@ public function ensureInt(
         $this[$key] = intval($val);
     }
 
+    /**
+     * Ensures that the value associated with the key is an int or null
+     */
+    public function ensureNullableInt(
```

Comment:
nit:

```suggestion
    public function ensureOptionalInt(
```

así es consistente con lis métodos de `\OmegaUp\Validators`